website_signup_legal_page_required/README.rst

```diff
@@ -68,6 +68,7 @@ Contributors
 * Jairo Llopis <yajo.sk8@gmail.com>
 * Alessio Gerace <alessio.gerace@agilebg.com>
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* José Luis Sandoval Alaguna <alagunasalahaddin@live.com>
```

Comment:
Do you prefer to not adopt the `readme` dir? https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

Reply:
I'm sorry, now i have no time to adopt the new format.